src/cart/test/util/cart_logtest.py

```diff
@@ -75,21 +76,80 @@ class LogError(LogCheckError):
 # error lines.
 shown_logs = set()
 
-# List of known free locations where there may be a mismatch, this is a
+# List of known locations where there may be a mismatch, this is a
 # dict of functions, each with a unordered list of variables that are
 # freed by the function.
 # Typically this is where memory is allocated in one file, and freed in
 # another.
-mismatch_free_ok = {'crt_plugin_fini': ('timeout_cb_priv',
-                                        'cb_priv',
-                                        'prog_cb_priv',
-                                        'event_cb_priv'),
-                    'crt_finalize': ('crt_gdata.cg_addr'),
-                    'crt_rpc_priv_free': ('rpc_priv'),
-                    'crt_grp_priv_destroy': ('grp_priv->gp_pub.cg_grpid',
-                                             'grp_priv->gp_psr_phy_addr'),
+mismatch_alloc_ok = {'crt_self_uri_get': ('tmp_uri'),
+                     'crt_rpc_handler_common': ('rpc_priv'),
+                     'crt_proc_d_iov_t': ('div->iov_buf'),
+                     'grp_add_to_membs_list': ('tmp'),
+                     'grp_regen_linear_list': ('tmp_ptr'),
+                     'crt_proc_d_string_t': ('*data'),
+                     'crt_hg_init': ('*addr'),
+                     'crt_proc_d_rank_list_t': ('rank_list',
+                                                'rank_list->rl_ranks'),
+                     'path_gen': ('*fpath'),
+                     'get_attach_info': ('reqb'),
+                     'iod_fetch': ('biovs'),
+                     'bio_sgl_init': ('sgl->bs_iovs'),
+                     'process_credential_response': ('bytes'),
+                     'pool_map_find_tgts': ('*tgt_pp'),
+                     'daos_acl_dup': ('acl_copy'),
+                     'dfuse_pool_lookup': ('ie', 'dfs', 'dfp'),
+                     'pool_prop_read': ('prop->dpp_entries[idx].dpe_str',
+                                        'prop->dpp_entries[idx].dpe_val_ptr'),
+                     'cont_prop_read': ('prop->dpp_entries[idx].dpe_str'),
+                     'cont_iv_prop_g2l': ('prop_entry->dpe_str'),
+                     'notify_ready': ('reqb'),
+                     'pack_daos_response': ('body'),
+                     'ds_mgmt_drpc_get_attach_info': ('body'),
+                     'pool_prop_default_copy': ('entry_def->dpe_str'),
+                     'daos_prop_dup': ('entry_dup->dpe_str'),
+                     'auth_cred_to_iov': ('packed')}
+
+mismatch_free_ok = {'crt_finalize': ('crt_gdata.cg_addr'),
```

Comment:
(pylint-duplicate-key) Duplicate key 'd_rank_list_free' in dictionary
(pylint-duplicate-key) Duplicate key 'get_tgt_rank' in dictionary